### PR TITLE
CA-313265, CA-314034: QEMU serial bug fixes

### PR DESCRIPTION
**Pull request by rosslagerwall**
Tested manually.

### CODE REVIEW

File: scripts/qemu-wrapper [hunk 1]
```diff
@@ -201,10 +204,18 @@ def main(argv):
             if params[0].startswith('tcp'):
                 print "serial: '%s'" % qemu_args[n + 1]
                 network_serial = True
+            elif params[0].startswith('file:'):
```
Comment (psafont) [inline scripts/qemu-wrapper]: I think using `type network_serial = Tcp | File | None` would be a better fit than having 2 seemingly independent boolean variables.

On line 118 onwards a match can be used instead.

And the comment doesn't seem relevant anymore ;)

Comment (rosslagerwall) [reply scripts/qemu-wrapper]: I don't understand. Python 2 doesn't have the sort of enum type you're referring to (short of defining your own class which IMO is overkill for this) nor does it have a match statement.

Which comment are you referring to that is no longer relevant?

Comment (psafont) [reply scripts/qemu-wrapper]: oof, I need some sleep: I thought the code was ocaml.

The comment is the one just before this if block, lines 202 and 203

Comment (rosslagerwall) [reply scripts/qemu-wrapper]: OK, I've dropped the comment.

**Pull request merged**